react-nextjs/react-host-nextjs-remote/e2e/CommonCheck.cy.ts

```diff
@@ -0,0 +1,39 @@
+import {BaseMethods} from "../../../cypress/common/base";
+import {baseSelectors} from "../../../cypress/common/selectors";
+import {Constants} from "../../../cypress/fixtures/constants";
+import base = Mocha.reporters.base;
+const basePage: BaseMethods = new BaseMethods()
+const appsData = [
+    {
+        appName: Constants.elementsText.reactHostNextJsRemoteHeader,
+        appNav: Constants.elementsText.reactHostNextJsRemoteNav,
+        host: 8080
+    }
+]
+
+appsData.forEach((
+    property: {
+        appName: string,
+        appNav: string,
+        host: number
+    }
+)=>{
+    describe(`Check ${property.appName} starts and running`, () => {
+        before(()=> {
+            basePage.openLocalhost(8080)
+        })
+        it(`Check Host ${property.appName} UI elements`, ()=> {
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.divElement,
+                text: property.appName
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.navigation,
+                text: property.appNav
+            })
+        })
+        it(`Check Remote ${property.appName} starts and running`, () => {
+            basePage.openLocalhost(8081)
+        })
```

Comment:
You can delete this `it` and update that one below to be:
```
    describe(`Check react (host) nextjs (remote) Apps`, () => {
        before(()=> {
            basePage.openLocalhost(property.host)
        })
        it(`Check ${property.appName} starts and running`, () => {
            basePage.openLocalhost(property.host)
            ... Also add there check of some elements exist or maybe that url contains localhost ....
        })
        if(property.host === ) {
            it(`Check ${property.appName} UI elements`, ()=> {
               basePage.checkElementWithTextPresence({
                   selector: baseSelectors.divElement,
                   text: property.appName
               })
               basePage.checkElementWithTextPresence({
                   selector: baseSelectors.navigation,
                   text: property.appNav
               })   
           }
        })
     })
```

Something like that